test/lib-faker.test.ts

```diff
@@ -52,6 +52,6 @@ test("lorem", (t) => {
 test("word", (t) => {
 	const faker = createFaker(t.title);
 
-	t.is(faker.word(), "amet,");
```

Comment:
having this test here makes me think that this was intentional (or at least known), although i don't understand why it was a good idea, what am i missing?

Reply:
Just a bad test, to be honest. It should be `amet` without the comma.